sys/sysmon/include/sysmon/ThreadStatus.hpp

```diff
@@ -47,12 +47,24 @@ class ThreadStatus: public MemoryStatus {
         /// Constructor
         ThreadStatus();
 
+        /// Increment thread loops number
+        inline void inc_loops(void) { loops_++; };
```

Comment:
Inline is implicit in C++ when the definition is present in the header file like in this case.
Moreover, inlining will never happen if the definition is in the source file.

Reply:
Done but should have been seen in previous PR. Painful and ugly to change all now...